torch/_dynamo/decorators.py

```diff
@@ -97,11 +97,13 @@ class set_stance(_DecoratorContextManager):
 
     _dynamo_forbidden = True
 
-    def __init__(self, stance: str, force_backend=None) -> None:
+    def __init__(
+        self, stance: str, skip_guard_eval_unsafe: bool = False, force_backend=None
```

Comment:
Let's force these to be kwargs:
```suggestion
        self, stance: str, *, skip_guard_eval_unsafe: bool = False, force_backend=None
```